Project.toml

```diff
@@ -15,7 +15,7 @@ Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 
 [compat]
-AWSCore = "0.5.5, 0.6"
+AWS = "1"
 AWSS3 = "0.6, 0.7, 0.8, 0.9"
```

Comment:
Should change these to `AWS.jl` compatible only versions.

Reply:
Done!